mail/templates/product_order_receipt/body.html

```diff
@@ -1,19 +1,44 @@
 {% extends "email_base.html" %}
 {% load format_discount %}
 
+{% block pre-logo-text %}
+<div class="rec-head" style="color: rgba(0,0,0,0.85); font: 14px/20px 'Arial', sans-serif; max-width: 696px; ">
+    <p style="font-weight: normal; margin: 0 0 20px;">Dear {{ purchaser.name }},</p>
+    <p style="font-weight: normal; margin: 0 0 20px;">You have been enrolled {% if content_title %} in {{ content_title }}{% endif %}.
+        The course should now appear on your MIT xPRO <a href="{{ base_url }}{% url 'user-dashboard' %}" style="color: #0070DA">dashboard</a>. You can also access your receipt by <a style="color: #0070DA" href="{{ base_url }}/receipt/{{order.id }}">clicking here</a>.
+    </p>
+    <p style="font-weight: normal; margin: 0 0 20px;">Below you will find a copy of your receipt:</p>
+
+</div>
+{% endblock %}
+
+{% block logo-address %}
+<div className="receipt-mit-info">
+    <p>
+      600 Technology Square
+      <br />
+      NE49-2000
+      <br />
+      Cambridge, MA 02139 USA
+      <br />
+      GSTIN: Pending
```

Comment:
This will be changed and loaded from the database once https://github.com/mitodl/mitxpro/pull/2832 is done